staging/src/k8s.io/csi-translation-lib/plugins/azure_disk.go

```diff
@@ -43,7 +43,7 @@ const (
 
 var (
 	managedDiskPathRE   = regexp.MustCompile(`.*/subscriptions/(?:.*)/resourceGroups/(?:.*)/providers/Microsoft.Compute/disks/(.+)`)
-	unmanagedDiskPathRE = regexp.MustCompile(`http(?:.*)://(?:.*)/vhds/(.+)`)
+	unmanagedDiskPathRE = regexp.MustCompile(`^http(?:.*)://(?:.*)/vhds/(.+)`)
```

Comment:
Is it possible to add unit tests for these?

Also cc @andyzhangx to double check azure syntax.

Reply:
lgtm on Azure part